src/main/java/com/maxmind/minfraud/response/Email.java

```diff
@@ -0,0 +1,46 @@
+package com.maxmind.minfraud.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This class contains minFraud response data related to the email address.
+ */
+public final class Email {
+    private final Boolean isFree;
+    private final Boolean isHighRisk;
+
+    public Email(
+            @JsonProperty("is_free") Boolean isFree,
+            @JsonProperty("is_high_risk") Boolean isHighRisk
+    ) {
+        this.isFree = isFree;
+        this.isHighRisk = isHighRisk;
+    }
+
+    public Email() {
+        this(null, null);
+    }
+
+    /**
+     * @return True if the email address is for a free email service provider.
+     */
+    public Boolean isFree() {
+        return isFree;
+    }
+
+    /**
+     * @return True if the email address is associated with fraud.
+     */
+    public Boolean isHighRisk() {
```

Comment:
I don't think these will be serialized correctly with the `toJson()` method as they are missing the `@JsonProperty("is_high_risk")`, etc., annotations.

I _think_ this is tested, but perhaps the new inputs did not end up in that test.